README.md

```diff
@@ -1,2 +1,227 @@
 # CapstoneProject
 This the Capstone Project of Module-1. I have created it with HTML, CSS and JavaScript. It is just like the actual project from a business customer. I strived to deliver beyond expectations.
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+ 
+  <br/>
+
+  <h3><b>Mike's Capstone Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone Project] <a name="about-project"></a>
+
+
+**[Capstone Project]** is the first milestone to get new Micronauts onboard with Figma, Own Responsive Website, Git, GitHub and Linters and how to fulfill the customer requirements.
```

Comment:
- [optional] you can add your site title instead of 'capstone project', also i can see you have two project title, one at the top and this one. Please, kindly use one for a professional README file and please ensure the description is about what you built.

Reply:
Dear Sir, thanks a lot for pointing it out for me, I have fixed it.